install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -1034,18 +1034,28 @@ hubble:
   # -- Unix domain socket path to listen to when Hubble is enabled.
   socketPath: /var/run/cilium/hubble.sock
 
-  # -- Configures the list of redact options for Hubble.
+  # -- Configures the redact options for Hubble.
   # Example:
   #
   #   redact:
-  #   - http-url-query
-  #   - kafka-api-key
+  #     enabled: true
+  #     http:
+  #       urlQuery: true
+  #     kafka:
+  #       apiKey: false
   #
-  # You can specify the list of options from the helm CLI:
+  # You can specify the options from the helm CLI:
   #
-  #   --set hubble.redact="{http-url-query,kafka-api-key}"
+  #   --set hubble.redact.enabled="true"
+  #   --set hubble.redact.http.urlQuery="true"
+  #   --set hubble.redact.kafka.apiKey="false"
   #
-  redact: ~
+  redact:
+    enabled: false
+    http:
+        urlQuery: false
+    kafka:
+        apiKey: false
```

Comment:
Yeap eventually we would like to have this behavior. I remember this was proposed at https://github.com/cilium/cilium/issues/23887#issuecomment-1614840591 to be fixed in a standalone PR. So how about dealing with those in a follow-up PR as proposed at https://github.com/cilium/cilium/issues/23887#issuecomment-1614840591? 

(we would need to maybe to discuss separately what the reasonable defaults would be?) 

Otherwise I can definitely apply the changes here :).